README.md

```diff
@@ -40,7 +40,17 @@ Make sure you have **Python 3.4** or better on your computer.
 
 To install pyno you must run ```pip install .``` from this directory.
 Pyno's dependencies `pyglet` and `pyperclip` are going to be installed automatically.
-Then run ```pyno``` in a console.
+In detail:
+```
+$ git clone https://github.com/honix/Pyno.git
+$ cd Pyno
+$ pip install .
+```
+alternatively as one-liner:
+```
+$ pip install git+https://github.com/honix/Pyno.git
+```
+Then run ```pyno``` in a console. The pip install step is optional and needed only to be able to run pyno from any working directory.
```

Comment:
> The pip install step is optional and needed only to be able to run pyno from any working directory.

I probably expressed myself badly. pip install is not optional (practically speaking), because this installs the "pyno" command that you use to run the program. 
If you don't do this, you have to open a python console, and then call it manually:
```
import pyno
pyno.app_run()
```

Reply:
Mentioned

Reply:
> python Pyno.py

still works from pyno clone directory, right? If yes my formulation was correct and more complete/detailed than the current one. If no we have to make it working again!

Reply:
> still works from pyno clone directory, right?

no, does not work. Why do we have to?